klaviyo/campaigns.py

```diff
@@ -0,0 +1,48 @@
+from .api_helper import KlaviyoAPI
+
+
+class Campaigns(KlaviyoAPI):
+    CAMPAIGNS = 'campaigns'
+    CAMPAIGN = 'campaign'
+    RECIPIENTS = 'recipients'
+
+    def get_campaigns(self, page=None, count=None):
+        """
+        Returns a list of all the campaigns you've created.
```

Comment:
this looks good but could you reformat the docstrings to keep it consistent?


Ex: https://github.com/klaviyo/python-klaviyo/blob/master/klaviyo/api_helper.py#L131 
```
def _
"""Handles the v2 api requests.

        Args:
            path (str): Url we make a request to.
            method (str): HTTP method.
            data (dict): Query parameters for the api call.

        Returns:
            (dict or arr): Response from Klaviyo API.
        """
```

Reply:
@remstone7 done